modules/ROOT/partials/auth/document-converters/php/initial-project-setup.adoc

```diff
@@ -0,0 +1,52 @@
+== Update PHP Configuration File
+
+Use the following command to locate the PHP configuration file:
+
+[source,bash]
+----
+php --ini
+----
+
+Open the configuration file in a text editor and ensure the following settings are enabled:
+
+[source,ini]
+----
+extension=openssl 
+extension_dir='ext'
+----
+
+The path to the extension directory may vary depending on your system.
```

Comment:
Perhaps a bit more emphasis could be drawn to this, so that it's harder to miss? Not sure if an Info tag would fit here, but this is fine too!

Reply:
```suggestion
[TIP]
The path to the extension directory may vary depending on your system.
```